src/App.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Movie } from './types/Movie';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filterFilms = (element: string, query: string): boolean => {
```

Comment:
I believe that the filterFilms() and getFilteredFilms() functions should be written inside a const App component: React.FC.
Both of these functions are inextricably linked to the component and will not be used without it.
It is interesting to know the opinion of the mentor.

Reply:
agree